spec/helpers/connections.rb

```diff
@@ -9,6 +9,22 @@ def local_connection
       Etcdv3.new(url: "http://#{local_url}")
     end
 
+    def local_connection_with_tls_server_auth(cacert)
+      Etcdv3.new(url: "http://#{local_url}", cacert: cacert)
+    end
+
+    def local_connection_with_tls_client_auth(cacert, key, cert)
+      Etcdv3.new(url: "http://#{local_url}", cacert: cacert, key: key, cert: cert)
```

Comment:
Just noticed that this should be https not http, but then all tests fail because the etcd test instance is not started with any tls parameters. I'm trying to geht this to work but that can take a while. i don't really grasp rspec and all this spec helper stuff yet...